e2etests/node_scan_rhcos_test.go

```diff
@@ -150,6 +151,9 @@ func TestGRPCGetRHCOSNodeVulnerabilities(t *testing.T) {
 	for name, c := range cases {
 		t.Run(name, func(t *testing.T) {
 			c := c
+			if !features.RHCOSNodeScanning.Enabled() {
```

Comment:
is this needed inside the loop or can this be checked at the very beginning of `func Test...`?